src/index.ts

```diff
@@ -936,7 +936,6 @@ class Envoy {
     log.normal("Opening a request (#%s) for Agent (%s) signature of payload: typeof '%s'", payload_id, agent_id, typeof payload);
     const event = `${agent_id}/wormhole/request`;
     log.silly(`Agent id: ${agent_id}`);
-    console.log("Event List: ", this.ws_server.eventList(this.opts.NS));
```

Comment:
Do we want to just make this a SILLY log instead of removing it entirely?

Reply:
it needs to be guaranteed to only take one line instead of 20 to be allowed to remain. I didn't want to deal with that so I just deleted it. wanna just leave it and someone can add it back the right way if they want it? we haven't had any bugs with event list